pages/account/index.tsx

```diff
@@ -1,18 +1,42 @@
 import Head from "next/head"
-import { Heading } from "@nypl/design-system-react-components"
+import { Heading, Text } from "@nypl/design-system-react-components"
 import Layout from "../../src/components/Layout/Layout"
 import initializePatronTokenAuth, {
   getLoginRedirect,
 } from "../../src/server/auth"
+import { MyAccountFactory } from "../../src/models/MyAccount"
+import type { Checkout, Hold, Patron, Fine } from "../../src/types/accountTypes"
 
-export default function MyAccount({ isAuthenticated }) {
+interface MyAccountProps {
+  checkouts: Checkout[]
+  holds: Hold[]
+  patron: Patron
+  fines: Fine
+  isAuthenticated: boolean
+}
+
+export default function MyAccount({
+  checkouts,
+  holds,
+  patron,
+  fines,
+  isAuthenticated,
+}: MyAccountProps) {
+  const sierraClientErrorThrown = !patron.name
+  console.log(checkouts, holds, patron, fines)
```

Comment:
This console.log can be removed

Reply:
It's there so we don't get a linting error for unused vars! also still being used generally for debugging.